lib/mix/nerves/utils.ex

```diff
@@ -195,16 +195,16 @@ defmodule Mix.Nerves.Utils do
   end
 
   def get_wsl_paths(file) do
-    {win_path, 0} = System.cmd("cmd.exe", ["/c", "cd"])
-    win_path = String.trim(win_path) <> "\\#{file}"
+    # Use wslpath, available from build Windows Build 17046
+    # https://superuser.com/questions/1113385/convert-windows-path-for-windows-ubuntu-bash
+    # -a    force result to absolute path format
+    # -u    translate from a Windows path to a WSL path (default)
+    # -w    translate from a WSL path to a Windows path
+    # -m    translate from a WSL path to a Windows path, with ‘/’ instead of ‘\\’
 
-    drive_letter =
-      Regex.run(~r/(.*?):\\/, win_path)
-      |> Enum.at(1)
-      |> String.downcase()
+    win_path = System.cmd("wslpath", ["-w -a", file])
```

Comment:
Did you test this? I'd think that this line should be `{win_path, 0} = System.cmd("wslpath", ["-w", "-a", file])`.